fix: strings with glob charaters were not being uploaded with wandb-core (#9475)
wandb/wandb: CHANGELOG.unreleased.md

Section headings should be at level 3 (e.g. `### Added`).

## Unreleased

> - fixed some charaters([?\*) would cause file uploads to fail with wandb-core (@jacobromero in https://github.com/wandb/wandb/pull/9475)

Reviewer: ```suggestion
- fixed some characters ([?\*) would cause file uploads to fail with wandb-core (@jacobromero in https://github.com/wandb/wandb/pull/9475)
```

- This changelog entry is not very clear.
- Could you specify that you fixed an issue with media not being handled properly when the file names included glob? characters.

> - fixed media file paths with glob characters ([?\*) would cause file uploads to fail with wandb-core (@jacobromero in https://github.com/wandb/wandb/pull/9475)

Reviewer: ```suggestion
- fixed an issue where media file paths containing special characters (?, *, ], [ or \) would cause file uploads to fail in `wandb-core`. (@jacobromero in https://github.com/wandb/wandb/pull/9475)
```
